src/index.ts

```diff
@@ -90,6 +90,7 @@ async function buildNuxt (options: StorybookOptions) {
     const plugins = await nuxtBuilder.normalizePlugins()
     generateStorybookFiles.call(nuxt.moduleContainer, {
       ...nuxtStorybookConfig,
+      nuxtOptions: { ...nuxt.options, build: undefined },
```

Comment:
Why build config should be removed?

Reply:
It causes circular reference

Reply:
Which code tries to serialize entire `nuxt.options`?

Reply:
This is wired, Just remove `build: undefined` and everything works fine.
Maybe I was doing something wrong at the time 😅